packages/rocketchat-apps/client/admin/appLogs.js

```diff
@@ -0,0 +1,73 @@
+import moment from 'moment';
+import hljs from 'highlight.js';
+
+Template.appLogs.onCreated(function() {
+	const instance = this;
+	this.id = new ReactiveVar(FlowRouter.getParam('appId'));
+	this.ready = new ReactiveVar(false);
+	this.app = new ReactiveVar({});
+	this.logs = new ReactiveVar([]);
+
+	const id = this.id.get();
+	const got = { info: false, logs: false };
+
+	RocketChat.API.get(`apps/${ id }`).then((result) => {
+		instance.app.set(result.app);
+
+		got.info = true;
+		if (got.info && got.logs) {
+			this.ready.set(true);
+		}
+	});
+
+	RocketChat.API.get(`apps/${ id }/logs`).then((result) => {
+		console.log('logs result:', result);
+
+		instance.logs.set(result.logs);
+
+		got.logs = true;
+		if (got.info && got.logs) {
+			this.ready.set(true);
+		}
+	});
```

Comment:
What happens if these both finish at the same time?  Maybe should wrap this in something that will catch both of these like Q.all ?

Because what if they finish at the same time I think extremely tiny chance, but might happen that neither execute: `this.ready.set(true)`